packages/p2p/src/components/floating-rate/floating-rate.jsx

```diff
@@ -0,0 +1,107 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { InputField, Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { localize } from 'Components/i18next';
+import { useStores } from 'Stores';
+import './floating-rate.scss';
+
+const FloatingRate = ({
+    change_handler,
+    className,
+    exchange_rate,
+    error_messages,
+    fiat_currency,
+    local_currency,
+    offset,
+    place_holder,
```

Comment:
```suggestion
    placeholder,
```

Reply:
since place_holder is a prop, it is in snake case

Reply:
Placeholder is a single word and hence it doesn't have to be in snake_case.

Reply:
Will make the changes